commandapi-platforms/commandapi-bukkit/commandapi-bukkit-test/commandapi-bukkit-test-tests/src/test/java/dev/jorel/commandapi/test/CommandUnregisterTests.java

```diff
@@ -118,7 +172,7 @@ public void setUp() {
 		// Set up a Vanilla command
 		vanillaResults = Mut.of();
 		new CommandAPICommand("test")
-			.withAliases("minecraft:test")
+			.withAliases("commandapitest:test")
```

Comment:
I think adding the namespaced version as an alias was a hack to get around the fact that the `minecraft` namespace isn't registered into the Brigadier dispatcher. Since other namespaces are put into the Brigadier dispatcher, we don't need to put it as an alias, and probably shouldn't since it causes the weird doubly-namespaced commands (`commandapitest:commandapitest:test`).